src/py42/services/trustedactivities.py

```diff
@@ -0,0 +1,68 @@
+from py42.exceptions import Py42BadRequestError
+from py42.exceptions import Py42DescriptionLimitExceededError
+from py42.exceptions import Py42HTTPError
+from py42.exceptions import Py42TrustedActivityConflictError
+from py42.exceptions import Py42TrustedActivityInvalidCharacterError
+from py42.services import BaseService
+
+
+class TrustedActivitiesService(BaseService):
+
+    _uri_prefix = "/api/v1/trusted-activities"
+
+    def __init__(self, connection):
+        super().__init__(connection)
+
+    def get_all(self, type=None):
```

Comment:
does this API not support paging? i.e. this endpoint always returns everything?

Reply:
I believe so? The format for the API response is 

`{
  "trustResources": [
    {
      "description": "string",
      "resourceId": "string",
      "type": "string",
      "updatedAt": "2021-09-14T17:35:10.070Z",
      "updatedByUserUid": "string",
      "updatedByUsername": "string",
      "value": "string"
    }
  ]
}`

I assumed if there's no page tokens then there's no paging? Or is that not the same thing?